python/paddle/static/nn/__init__.py

```diff
@@ -63,7 +63,6 @@
     'fc',
     'batch_norm',
     'embedding',
-    'bilinear_tensor_product',
```

Comment:
Currently, `paddle.static.nn.bilinear_tensor_produc` is public API, so can NOT be delete from \_\_all\_\_, otherwise will be incompatible upgrade.

Reply:
> Currently, `paddle.static.nn.bilinear_tensor_produc` is public API, so can NOT be delete from __all__, otherwise will be incompatible upgrade.

The bilinear_tensor_produc is put back into __all__ list.